source/03-specifications/02-formats-for-counter-reports.rst

```diff
@@ -9,21 +9,21 @@ Formats for COUNTER Reports
 
 R5 reports can be delivered in tabular form, or as machine-readable data (JSON) via the COUNTER_SUSHI API. The tabular form MUST be provided as either an Excel or a tab-separated-value (TSV) file, or both. Additional file formats that can be easily imported into spreadsheet programs without loss or corruption may be offered at the vendor's discretion. The reports in JSON, TSV and other text formats MUST be encoded using UTF-8. The JSON format MUST comply with the COUNTER_SUSHI API Specification (see :numref:`sushi` below).
 
-All COUNTER reports have the same layout and structure. Figure 3.b (below) provides an example of the “Journal Requests (Controlled)” Standard View. Figure 3.c (below) shows the layout for tabular reports, which will be the focus of the discussions throughout this document. Note that the COUNTER_SUSHI API Specification includes the same elements with the same or similar names; therefore, understanding the tabular reports translates to an understanding of what is REQUIRED in reports retrieved via the COUNTER_SUSHI API.
+All COUNTER reports have the same layout and structure. Figure 3.b (below) provides an example of the Title Report. Figure 3.c (below) shows the layout for tabular reports, which will be the focus of the discussions throughout this document. Note that the COUNTER_SUSHI API Specification includes the same elements with the same or similar names; therefore, understanding the tabular reports translates to an understanding of what is REQUIRED in reports retrieved via the COUNTER_SUSHI API.
 
-.. figure:: ../_static/img/3b-standard-view-sample.png
-   :alt: Standard View sample
+.. figure:: ../_static/img/Figure-3b.png
+   :alt: Title Report sample
    :align: center
    :width: 80%
 
-.. centered:: Figure 3.b: Sample “Journal Requests (Controlled)” Standard View
+.. centered:: Figure 3.b: Sample Title Report
 
-.. figure:: ../_static/img/3c-tabular-report-layout.png
-   :alt: Tabular Report layout
+.. figure:: ../_static/img/Figure-3c.png
```

Comment:
Please replace "Report / Item" (I have no idea why that term was used in the old image) with "Report Attributes". The singular/plural should be consistent (e.g. Metric types but Reporting period total). I would prefer the same upper/lower case as in the text, e.g. Metric Types.

Reply:
I forgot to mention that I would prefer vertical dotted line between the areas as in the old image.